plugins/luxtronic2/__init__.py

```diff
@@ -328,22 +399,44 @@ def _decode(self, identifier, value):
         return value
 
     def parse_item(self, item):
+        def __reverseListOp(param):
+            # reversing list building of smarthome.py config parser for the
+            # binary or opperator |
```

Comment:
I did not know that quotes are supported, but if it works it's most definitely a better way to do it. I am going to fix it when there is time for it.